app/views/projects/members/index.html.erb

```diff
@@ -26,17 +35,17 @@
             "
           >
             <% @members.each do |member| %>
+              <% member_source = Member.membership_source(@namespace, member.user) %>
```

Comment:
Due to comments above, this will need to pass `member` instead of `member.user`

Reply:
Updated in 0379505201abd0d8b0a317f46c1819e1c82f5c3e